src/util/string.h

```diff
@@ -519,6 +519,39 @@ std::basic_string<T> unescape_enriched(const std::basic_string<T> &s)
 	return output;
 }
 
+template <typename T>
+std::vector<std::basic_string<T> >
+	split(const std::basic_string<T> &s, T delim)
+{
```

Comment:
Oops, I had forgotten to delete that one. It is exactly the same function, except that it accepts any kind of string.